s3backgrounddelete/s3backgrounddelete/s3backgroundconsumer

```diff
@@ -4,6 +4,16 @@ import sys
 import os
 from s3backgrounddelete.object_recovery_processor import ObjectRecoveryProcessor
 
+if (len(sys.argv)>1):
```

Comment:
We should use the python command line parser instead of replying on argument sequence